vembrane/common.py

```diff
@@ -42,27 +43,25 @@ def normalize(s: str) -> str:
     return shlex.quote(swap_quotes(s) if not single_outer(s) else s)
 
 
-def get_annotation_keys(header: VariantHeader, ann_key: str) -> List[str]:
+def get_annotation_keys(header: VCFHeader, ann_key: str) -> List[str]:
     separator = "'"
-    for rec in header.records:
-        if rec.key == "VEP":
-            separator = ":"
-            continue
-        if rec.get("ID") == ann_key:
-            return list(
-                map(
-                    str.strip,
-                    rec.get("Description").strip('"').split(separator)[1].split("|"),
-                )
+    if header.contains_generic("VEP"):
+        separator = ":"
+    if h := header.infos.get(ann_key):
+        return list(
+            map(
+                str.strip,
+                h.get("Description").strip('"').split(separator)[1].split("|"),
```

Comment:
I always felt like this method seems a bit brittle, but it works, so it's fine ;D
However, we do not check anywhere for multiple different annotation keys, e.g. if snpeff or something else adds a CSQ or EFF annotation entry etc.

Reply:
Unfortunately there are also other tools that annotate in a similar way. I would like to move the discussion about that to the follow up PR, where I introduce a way to use user defined custom annotations by an external provided yaml.

Reply:
Sure, that makes sense